### PR TITLE
Translation Completed v1 (with Korean)

### PR DESCRIPTION
**Pull request by newtype0096**
Hello :)
I finished translation this app with Korean (ko_KR)
Please check and tell me your idea :D

### CODE REVIEW

File: Localization/de_DE/Localizable.strings [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*
+﻿/*
```
Comment (HNC-77) [inline Localization/de_DE/Localizable.strings:1]: HNC-77 amail:Salmantoma2@gmail.com

**Review by HNC-77 — APPROVED** [review]

**Comment by YaPSNapp** [comment]
Thanks a lot. I will check this soon to include it in next release. Thanks again

File: Localization/ko_KR/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,586 @@
...
+"LEVEL_UP" = "레벨 업!";
+"LEVEL_UP_DESC" = "LEVEL %hd을 축하합니다 %@ !";
+"CONGRATS" = "축하합니다 !";
+"NEW_PLATINUM" = "새로운 플래티넘 !";
+"WN_THANKS" = "**감사합니다 !!**";
+"WN_THANKS_STITLE" = "지원 감사합니다 !";
+"WN_THANKS_DESC" = "풀 버전을 구입했습니다, 새로운 기능을 확인해 봅시다.";
+"WN_FULLVERSION_TITLE" = "**새로운 기능**";
+"WN_FULLVERSION_STITLE" = "";
+"WN_FULLVERSION_DESC" = "이제 새로운 기능을 이용할 수 있습니다.\nPSN 메시지 (YaPSNapp 풀 버전 사용자간 즉시 알림), 광고 없음, 요청 무제한, 게임 세션 기록 등";
```
Comment (HNC-77) [inline Localization/ko_KR/Localizable.strings:579]: HNC-77

File: Localization/de_DE/Localizable.strings [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/*
```
Comment (HNC-77) [inline Localization/de_DE/Localizable.strings:1]: amil:Salmantoma2@gmail.com

File: Localization/de_DE/Localizable.strings [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
+﻿/*
  Localizable.strings
```
Comment (HNC-77) [inline Localization/de_DE/Localizable.strings:2]: amil:Salmantoma2@gmail.com

File: Localization/de_DE/Localizable.strings (hunk 3 above)
Comment (HNC-77) [reply Localization/de_DE/Localizable.strings:1]: Playstation.com

**Review by HNC-77 — COMMENTED** [review]
ID:HNC-77 PS4 GTA5

**Pull request merged**